### PR TITLE
[MM-33]: Added the test cases for the tooltip upon hovering the cursor over the issue link in the channel or DM/GM on MM.

### PR DESCRIPTION
**Pull request by arush-vashishtha**
### Summary
This PR consists of the test cases for the tooltip upon hovering the cursor over the issue link in the channel or DM/GM on MM.

### CODE REVIEW

File: data/test-cases/plugins/jira/miscellaneous/cloud/Tooltip_over_link.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Connect your jira account on you MM account and refresh the page.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/jira/miscellaneous/cloud/Tooltip_over_link.md]:
```suggestion
1. Connect your jira account with your MM account and refresh the page.
```

File: data/test-cases/plugins/jira/miscellaneous/cloud/Tooltip_over_link.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+2. Hover the cursor over any issue link present in any desired channel or DM/GM.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/jira/miscellaneous/cloud/Tooltip_over_link.md]:
```suggestion
2. Hover the cursor over any Jira issue link present in any desired channel or DM/GM.
```

File: data/test-cases/plugins/jira/miscellaneous/cloud/Tooltip_over_link.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+**Step 2**
+
+1. Edit the issue whose link is already present in any desired channel or DM/GM on MM.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/jira/miscellaneous/cloud/Tooltip_over_link.md]:
```suggestion
1. Edit the issue on Jira whose link is already present in any desired channel or DM/GM on MM.
```

File: data/test-cases/plugins/jira/miscellaneous/cloud/Tooltip_over_link.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+2. Navigate to the channel where the link already exists and refresh the page.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/jira/miscellaneous/cloud/Tooltip_over_link.md]:
```suggestion
2. Navigate to the channel on MM where the link already exists and refresh the page.
```

File: data/test-cases/plugins/jira/miscellaneous/cloud/Tooltip_over_link.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+3. Hover the cursor over the edited issue link in the desired channel or DM/GM on MM.
+
+**Step 3**
+
+1. Delete the issue whose link is already present in any desired channel or DM/GM on MM.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/jira/miscellaneous/cloud/Tooltip_over_link.md]:
```suggestion
1. Delete the issue on Jira whose link is already present in any desired channel or DM/GM on MM.
```

File: data/test-cases/plugins/jira/miscellaneous/server/Tooltip_over_link.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
+---
+# (Required) Ensure all values are filled up
+name: "Tooltip over link is showing correct ticket information on the server instance."
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/jira/miscellaneous/server/Tooltip_over_link.md]: Please fix the above comments here also, and also add the Jira server or Jira cloud wherever specifying any task which would be completed on Jira.

Comment (arush-vashishtha) [reply data/test-cases/plugins/jira/miscellaneous/server/Tooltip_over_link.md]: Where ever i have edited a issue and deleted a issue i have added the respective cloud and server.

**Review by AayushChaudhary0001 — APPROVED** [review]

**Review by raghavaggarwal2308 — APPROVED** [review]

**Pull request merged**